source/Logging/source/RequestResponseMiddleware/RequestResponseMiddleware.csproj

```
@@ -26,7 +26,7 @@ limitations under the License.
 
   <PropertyGroup>
     <PackageId>Energinet.DataHub.Core.Logging</PackageId>
-    <PackageVersion>2.2.8$(VersionSuffix)</PackageVersion>
+    <PackageVersion>3.0.0$(VersionSuffix)</PackageVersion>
```

Is it a breaking change? How about?
```suggestion
    <PackageVersion>2.3.0$(VersionSuffix)</PackageVersion>
```

---

It is not. I will bump the version to `2.3.0` instead.